crux-core/src/crux/kv/index_store.clj

```diff
@@ -452,22 +460,77 @@
 (defn- value-buffer->id-buffer [index-snapshot ^DirectBuffer value-buffer]
   (c/->id-buffer (db/decode-value index-snapshot value-buffer)))
 
+(defn- ecav-cache-lookup ^java.util.Map [ecav-cache ecav-i ^DirectBuffer eid-value-buffer ^DirectBuffer content-hash-buffer]
```

Comment:
mind if we call this `cav-cache` given we don't store the `e`?

Reply:
Good call, will do.